src/lib/src/select/select.component.ts

```diff
@@ -103,6 +104,7 @@ export class TsSelectComponent extends TsReactiveFormBaseComponent {
 
   /**
    * Define the key that represents the value item from the object
+   * If not provided, currently it defaults to key "name"
```

Comment:
Let's add a line break here.

When our docs are generated these two lines will be combined into a single line unless there is an empty line separating them (which is a good thing for something like lines 130+131).